btree/btree.c

```diff
@@ -1641,10 +1672,18 @@ static void bnode_make(struct slot *slot)
 	slot->s_node->n_type->nt_make(slot);
 }
 
-static void bnode_val_resize(struct slot *slot, int vsize_diff)
+static bool bnode_newval_isfit(struct slot *slot, struct m0_btree_rec *old_rec,
```

Comment:
add a function declaration for this function after line 1239.
It wont affect compilation, but we have done it for all the bnode functions.